### ISSUE TITLE
ngx-input-star-rating - a11y

### ISSUE THREAD

**Issue by coryrylan** [coryrylan/ngx-libraries#1]
The buttons on the rating component are missing the appropriate aria-label attributes.

**Comment by coryrylan** [comment]
fixed https://github.com/coryrylan/ngx-libraries/commit/53645fec6074d82fc5629e152d3a78f677244097
